module/fastgltf.cppm

```diff
@@ -0,0 +1,166 @@
+module;
+
+#include <cassert>
```

Comment:
Is this a bug with the core header not including `cassert` itself?

Reply:
core.hpp includes type.hpp, which uses assert macro. Since a module does not export macro, <cassert> have to be manually included in global module fragment.

Reply:
But wouldn't that only apply to when we use `import std;` then, which wouldn't export the macro? `cassert` is included normally from `types.hpp`.

Reply:
What you say is if FASTGLTF_USE_STD_MODULE macro not defined, cassert header is already included in types.hpp, therefore it doesn't need to be included in the global module fragment? Am I understanding right?

Reply:
Yeah. I am currently writing some CMake stuff locally to support this addition out-of-the-box which I will push later onto your fork, and I have removed it locally and that doesn't cause any issues.